tasks/system_probe.py

```diff
@@ -198,9 +198,6 @@ def test(ctx, skip_object_files=False, only_check_bpf_bytes=False, bundle_ebpf=T
         # bpf_tag += ",ebpf_bindata"
         cmd += " -run=TestEbpfBytesCorrect"
     else:
-        if getpass.getuser() != "root":
-            print("system-probe tests must be run as root")
-            raise Exit(code=1)
```

Comment:
Do our tests no longer require root ?  This was added because the conntrack/ebpf tests fail in mysterious ways when not run as root.